feature_layers/feature-collection-layer-from-portal/README.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "category": "Feature layers",
+    "description": "Combine feature tables with different geometries into a single layer.",
+    "ignore": false,
+    "images": [
+        "FeatureCollectionLayer.png"
+    ],
+    "keywords": [
+        "FeatureCollection",
+        "FeatureCollectionLayer",
+        "FeatureCollectionTable",
+        "Feature",
+        "Field",
+        "SimpleFillSymbol",
+        "SimpleLineSymbol",
+        "SimpleMarkerSymbol",
+        "SimpleRenderer"
+    ],
+    "redirect_from": "/java/latest/sample-code/feature-collection-layer.htm",
```

Comment:
I _think_ we leave this blank for new samples, and it gets automatically populated when the developer webpage is generated. I'll double check!

Reply:
This should only be here if this sample is meant to replace the feature collection layer sample.